Esri/arcgis-maps-sdk-unreal-engine-samples
# Split samples into general samples project and xr samples project

**Sample**

Split Samples Project

**Summary**

- Created a new project explicitly for XR samples in Unreal Engine
- Moved GitIgnore to Parent Directory in the project
- Added necessary lines in GitIgnore to ignore the same files in the xr samples project as in the samples project

**How to Test**

- Clone the branch *Mas12517/split project*
- Navigate to repository directory in File Explorer
- Open the folder labeled xr_sample_project
- Right click on xr_samplesproject.uproject and click "Generate Visual Studio Project Files"
- Once complete open git and ensure that there are no new files are being tracked
- Open the project

**Tests**

Windows, will test mac in office

**ArcGIS Maps SDK Version**

N/A & Unreal 5.3.1
